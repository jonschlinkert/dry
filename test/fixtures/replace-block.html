{% extends "basic" %}
{% block replace="header" %}
header: "replace-block.html"
{% endblock %}