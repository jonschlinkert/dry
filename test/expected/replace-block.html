
unique: "blocks/basic.html"

header: "replace-block.html"
